docs/userguide.md

```diff
@@ -271,12 +271,12 @@ kubectl patch clusterrole kubefed-role --type='json' -p='[{"op": "add", "path":
         }
 }]'
 ```
-This example is for cluster scoped federation deployment. For namespaced federation deployment,
-you can patch role `kubefed-role` in the kubefed namespace instead.
+This example is for cluster scoped KubeFed deployment. For namespaced KubeFed deployment,
+you can patch role `kubefed-role` in the `kubefed` namespace instead.
 
 ### Disabling federation of an API type
 
-You can disable propagation of an API type using the
+You can disable federation of an API type using the
```

Comment:
Please revert - propagation is the appropriate terminology here.  The header should be updated instead.

Reply:
Also, the docs below are incorrect.  `kubefedctl disable` removes the `FederatedTypeConfig` (and optionally the crd).  The user has to edit a `FederatedTypeConfig` to disable propagation.

@font Given that you made the changes to the disable behavior, I'd appreciate it if you would update the documentation to match the current behavior.

Reply:
Thanks - I had done this but then as I went further through the doc I wasn't sure if we wanted to replace the term 'federation' with 'propagation' completely so I undid the changes so that I could check this first. I will go ahead and update this section to 'propagation'.

Reply:
> Also, the docs below are incorrect. `kubefedctl disable` removes the `FederatedTypeConfig` (and optionally the crd). The user has to edit a `FederatedTypeConfig` to disable propagation.
> 
> @font Given that you made the changes to the disable behavior, I'd appreciate it if you would update the documentation to match the current behavior.

Addressed in #924.